packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -89,7 +89,7 @@ export const globalOptions = {
     ),
     expand: async (publicKeys: string) => {
       return publicKeys
-        .split(',')
+        ?.split(',')
```

Comment:
how come it was able to be undefined? validation schema also is `z.string()` so this should not be possible right

Reply:
it's an optional field so it could be undefined